.github/workflows/e2e-tests.yml

```diff
@@ -98,12 +98,16 @@ jobs:
     - name: Run all tests
       if: matrix.gutenberg == 'no'
       working-directory: projects/plugins/jetpack/tests/e2e
-      run: yarn test-e2e
+      run: yarn test-e2e --testPathIgnorePatterns=search
 
     - name: Run only blocks tests with latest Gutenberg
       if: matrix.gutenberg == 'latest'
       working-directory: projects/plugins/jetpack/tests/e2e
       run: yarn test-e2e --testNamePattern=blocks
+    - name: Run only search tests
```

Comment:
Since these tests are running together with `Run all tests`, overall job time is about 17minutes. We probably should bump `timeout-minutes` value to add a bit of slack there.

Reply:
bumped `timeout-minutes` to 25 in https://github.com/Automattic/jetpack/pull/19447/commits/a1377a85dba2c73b96966eccca55bfae26cdb1fd